pagopa/selfcare-onboarding
# [SELC-3158] feat: send mail async with contract

#### List of Changes



* Added a new asynchronous step to the workflow.
* Integrated functionality to trigger email delivery during onboarding.
* Implemented the attachment of the organization's contract to the outgoing email.

#### Motivation and Context



This pull request introduces a enhancement by implementing a new asynchronous step in the workflow. Specifically, it enables the seamless initiation of an email dispatch during the onboarding of an organization from the frontend, along with the attachment of the corresponding contract.

#### How Has This Been Tested?






1. Navigate to the organization onboarding section in the frontend.
2. Complete the onboarding process for a test organization.
3. Verify that the email notification is sent promptly.
4. Check the email content and confirm the correct attachment of the organization's contract.


#### Screenshots (if appropriate):

#### Types of changes




#### Checklist:
